src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeGroupController.php

```diff
@@ -79,12 +83,36 @@ public function indexAction(Request $request, AttributeGroupFilters $attributeGr
      *     message="You do not have permission to create this."
      * )
      *
-     * @return RedirectResponse
+     * @param Request $request
+     *
+     * @return Response
      */
-    public function createAction()
+    public function createAction(Request $request): Response
     {
-        //@todo: implement in antoher pr
-        return $this->redirectToRoute('admin_attribute_groups_index');
+        $attributeGroupFormBuilder = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\AttributeGroupFormBuilder');
+        $attributeFormHandler = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\AttributeGroupFormHandler');
```

Comment:
But does are not actual classes just service names, for example
``` PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\AttributeGroupFormBuilder:
    autowire: true
    class: 'PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\FormBuilder'
    factory: [ '@prestashop.core.form.builder.form_builder_factory', 'create' ]
    arguments:
      - 'PrestaShopBundle\Form\Admin\Sell\Catalog\AttributeGroupType'
      - '@PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider\AttributeGroupFormDataProvider'```

Reply:
Yes but the advantage of using FQCN names for the services is that you can use this `MyClass::class` as the service name since they should match, it's more elegant this way and it simplifies refacto as well since you can easily find the usages of the service